interface/wx/math.h

```diff
@@ -89,8 +89,14 @@ double wxRadToDeg(double rad);
 unsigned int wxCTZ(wxUint32 x);
 
 /**
-    Small wrapper around round().
+    Small wrapper around std::lround().
+
+    This function exists for compatibility, as it was more convenient than
+    std::round() before C++11. Use std::lround() in the new code.
+
+    It is defined for all integer and floating point types @c T.
```

Comment:
Since you are deprecating the integer variants, probably shouldn't mention them here

Reply:
Right, thanks.